service/kvmconfig/resource_router_test.go

```diff
@@ -6,18 +6,19 @@ import (
 	"testing"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
-	"github.com/giantswarm/kvm-operator/service/resource/configmapv3"
-	"github.com/giantswarm/kvm-operator/service/resource/servicev2"
 	"github.com/giantswarm/operatorkit/framework"
+
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v2/resource/configmap"
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v2/resource/service"
 )
 
 func Test_Service_newResourceRouter(t *testing.T) {
 	resourceV1 := []framework.Resource{
-		&servicev2.Resource{},
+		&service.Resource{},
```

Comment:
That does not make sense somehow. Here we use the v2 service and configmap resource but above we use the v3 key package. WDYT?

Reply:
I think it's weird we'll have to change the test as we add and remove versions. I think it'd be nice to add resource fixture here.